### PR TITLE
kubernetes: Use the nav-pf-vertical sidebar styling

### PR DESCRIPTION
**Pull request by stefwalter**
In addition resize the sidebar as the height and width of the screen change.

Depends on #4213

### CODE REVIEW

**Comment by stefwalter** [comment]
screenshot from 2016-04-15 08-28-08
screenshot from 2016-04-15 08-28-11
screenshot from 2016-04-15 08-28-14

**Comment by andreasn** [comment]
This looks great!

**Comment by andreasn** [comment]
Not sure if this came with this pull, or if it's just easier to see now, but there is less padding on the left than of the right for the content of each page.

screenshot from 2016-04-15 18-55-42

**Comment by andreasn** [comment]
But I can file that as a separate issue. Apart from that, looks good to me!

**Comment by andreasn** [comment]
This is current master. The left margin is indeed smaller.

screenshot from 2016-04-18 13-50-19

**Comment by stefwalter** [comment]
This is ready to merge.

**Comment by stefwalter** [comment]
Priority because the nodes stuff just merged needs this change in order to not break navigation.

File: pkg/kubernetes/index.html [hunk 1]
```diff
@@ -63,53 +63,62 @@
...
+                </li>
+                <li class="list-group-item" ng-class="{ active: viewActive('projects')}"
+                    ng-if="settings.flavor == 'openshift'">
+                    <a ng-href="#{{ viewUrl('projects') }}">
+                        <i class="pficon pficon-project fa-fw" title="Projects"></i>
+                        <span class="list-group-item-value" translatable="yes">Projects</span>
+                    </a>
+                </li>
+            </ul>
+>>>>>>> f35db93... kubernetes: Use the nav-pf-vertical sidebar styling
```
Comment (dperpeet) [inline pkg/kubernetes/index.html]: I think this slipped in by accident

**Comment by dperpeet** [comment]
I'll do a fixup while merging

**Pull request closed without merging**